ui/src/translations/fr.json

```
@@ -135,5 +136,10 @@
   "please_refer_to": "Veuillez vous référer à",
   "import_solution_hint": "Vous devez importer une solution pour préparer votre environnement",
   "redirecting": "Redirection…",
-  "documentation": "Documentation"
+  "documentation": "Documentation",
+  "env_preparation": "Environment preparation",
+  "env_preparation_success": "L'environnement {envName} est en cours de préparation.",
+  "env_preparation_failed": "La préparation de l'environnement {envName} a échoué.",
+  "preparing_environemnt": "préparer l'environnement",
```

If this is supposed to mean the same thing as the english variant, it should translate to:

```suggestion
  "preparing_environemnt": "environnement en cours de préparation",
```